cardano-api/src/Cardano/Api/Protocol.hs

```diff
@@ -117,3 +120,9 @@ mkNodeClientProtocol protocol =
           -- client case.
           (EpochSlots 21600)
           (Consensus.SecurityParam 2160)
+
+data ProtocolData
```

Comment:
Why didn't we change the `Protocol` type in this same module, so that `mkNodeClientProtocol` would do the right thing?